__tests__/types.d.ts

```diff
@@ -89,3 +103,20 @@ export interface AppSettings {
   deleteAccount: () => void;
   logout: () => void;
 }
+
+export interface Shelter {
+  id: number;
+  name: string;
+  associates: User[];
+  permissionRequests: PermissionRequest[];
+  photoAlbum: PhotoAlbum;
+  contactInfo: ContactInfo;
+  removePermission: (permission: Persmission, user: User) => void;
+  resolveRequest: (request: PermissionRequest) => void;
+  modifyAssociate: (associate: User, permission: Permission) => void;
```

Comment:
We probably don't want to specify `permission` as an argument since we are about to modify certain user.